src/main/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaEmailClient.java

```diff
@@ -6,6 +6,7 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.strikeoffobjections.common.AvroSerializer;
```

Comment:
this import has been added but nothing in the class has changed, is it needed?

Reply:
yeah, the AvroSerializer was in the same package previously so no need to import, it's now moved to common